src/invert/laplace/impls/mumps/mumps_laplace.cxx

```diff
@@ -211,21 +211,21 @@ LaplaceMumps::LaplaceMumps(Options *opt, const CELL_LOC loc, Mesh *mesh_in = mes
     localrhssize += localmesh->xstart*(localmesh->LocalNz);
     
     int nxpe = localmesh->NXPE;
-    localrhs_size_array = Array<int>(nxpe);
+    localrhs_size_array.resize(nxpe);
```

Comment:
reallocate?

Reply:
Oops, thanks! Due to me relying on the compiler and not actually compiling everything. Luckily we have code review! :smile: